### PR TITLE
Ensure that map.vue does not fail if GeoJSON has no property

### PR DESCRIPTION
**Pull request by noirbizarre**
When using the geom field from spatial coverage (manualy submited coverage zone), the map.vue fails because not properties are present in the GeoJSON.
This PR avoid this by checking the feature presence.

It raises a question: what do we chose to display when we have both `geom` and `zones` in the spatial coverage. The current behavior is to display the manualy submitted `geom`.

### CODE REVIEW

File: js/components/widgets/map.vue [hunk 1]
```diff
@@ -65,29 +65,23 @@ export default {
...
-                        layer.on("mouseout", function () {
-                            layer.closePopup();
-                        });
+                        if (feature.properties && feature.properties.name) {
+                            layer.bindPopup(feature.properties.name);
+                            layer.on("mouseover", function () {
+                                layer.openPopup();
+                            });
+                            layer.on("mouseout", function () {
+                                layer.closePopup();
```
Comment (davidbgk) [inline js/components/widgets/map.vue]: Yay! A conflict with my branch :p

I think you should remove the `closePopup` in that PR as it doesn't exist in leaflet and raises an error.

**Comment by davidbgk** [comment]
> what do we chose to display when we have both geom and zones in the spatial coverage. The current behavior is to display the manualy submitted geom.

Maybe we should warn the user and tell him that we'll use our own zones because it's updated etc?

**Comment by davidbgk** [comment]
r+wc

File: js/components/widgets/map.vue (hunk 1 above)
Comment (noirbizarre) [inline js/components/widgets/map.vue]: I didn't to touch the popup behavior as it has been handled in a separate issue but if you prefer, I can remove it.

File: js/components/widgets/map.vue [hunk 2]
```diff
@@ -65,29 +65,29 @@ export default {
...
-                        layer.on("mouseover", function () {
-                            layer.openPopup();
-                        });
-                        layer.on("mouseout", function () {
-                            layer.closePopup();
-                        });
+                    onEachFeature: (feature, layer) => {
+                        if (feature.properties && feature.properties.name) {
+                            layer.bindPopup(feature.properties.name);
+                            layer.on("mouseover", function() {
```
Comment (davidbgk) [inline js/components/widgets/map.vue]: `=>` here and below? :)

File: js/dataset/display.js [hunk 3]
```diff
@@ -165,7 +165,13 @@ function load_coverage_map() {
                 layer.openPopup();
             });
             layer.on('mouseout', function() {
```
Comment (davidbgk) [inline js/dataset/display.js]: Same here

**Comment by noirbizarre** [comment]
OK, I fixed the close popup instead. Good to go ?

**Pull request merged**